dev/integration_tests/android_embedding_2/README.md

```diff
@@ -0,0 +1,16 @@
+# android_embedding_2
```

Comment:
nit: I would just delete this if it's not going to have any unique information.

Reply:
I added some relevant information :)